mappings/net/minecraft/server/command/ServerCommandSource.mapping

```diff
@@ -5,14 +5,37 @@ CLASS cd net/minecraft/server/command/ServerCommandSource
 	FIELD d position Lcro;
 	FIELD e world Lvf;
 	FIELD f level I
-	FIELD g name Ljava/lang/String;
-	FIELD h textComponent Ljm;
+	FIELD g simpleName Ljava/lang/String;
+	FIELD h nameTextComponent Ljm;
 	FIELD i minecraftServer Lnet/minecraft/server/MinecraftServer;
 	FIELD j silent Z
 	FIELD k entity Laid;
 	FIELD l resultConsumer Lcom/mojang/brigadier/ResultConsumer;
 	FIELD m entityAnchor Lck$a;
 	FIELD n rotation Lcrn;
+	METHOD <init> (Lcc;Lcro;Lcrn;Lvf;ILjava/lang/String;Ljm;Lnet/minecraft/server/MinecraftServer;Laid;)V
+		ARG 1 output
+		ARG 2 pos
+		ARG 3 rot
+		ARG 4 world
+		ARG 5 level
+		ARG 6 simpleName
+		ARG 7 nameTextComponent
+		ARG 8 server
+		ARG 9 entity
+	METHOD <init> (Lcc;Lcro;Lcrn;Lvf;ILjava/lang/String;Ljm;Lnet/minecraft/server/MinecraftServer;Laid;ZLcom/mojang/brigadier/ResultConsumer;Lck$a;)V
+		ARG 1 output
+		ARG 2 pos
+		ARG 3 rot
+		ARG 4 world
+		ARG 5 level
+		ARG 6 rsimpleName
```

Comment:
is the `r` prefix here intended?

Reply:
Enigma just keeps writing r in the name field when I press r to rename.